src/components/Modules/Home/HomeContentModule.tsx

```diff
@@ -1,7 +1,24 @@
+import Sidebar from "~/components/SideBar.tsx/Sidebar";
+import HeroSectionModule from "../HeroSection/HeroSectionModule";
+
+
 type Props = {
-  title: string;
-};
+  heroSectionModuleProps: React.ComponentProps<typeof HeroSectionModule>
+}
 
 export const HomeContentModule = (props: Props) => {
-  return <div>{props.title}</div>;
+  return (
+    <div className='mx-auto max-w-[1300px] py-6'>
+      <div className='mb-8 flex gap-4 text-7xl font-bold'>
+        <h1>Christina</h1>
+        <h1 className='font-outline-2 text-white'>Gray</h1>
+      </div>
+      <div className='grid grid-cols-4 gap-12'>
+        <div className='col-span-1'>
+          <Sidebar />
+        </div>
+        <HeroSectionModule {...props.heroSectionModuleProps } />
```

Comment:
Don't use Modules in Module code
[Check Module Convention Docs](https://coal-blouse-907.notion.site/20-4-Atomic-Design-Pattern-Module-tsx-11a268a31d0a4a32afdd811b691024a4?pvs=4)

Reply:
Thank you so much @kiteprize